planet/src/main/java/com/google/planet/data/Event.java

```diff
@@ -45,5 +47,15 @@ public String getAddress(){
     public TimeRange getOpeningHours(){
         return this.openingHours;
     }
+
+    /**
+    * A comparator for sorting events by the duration of their opening hours, in ascending order.
+    */
+    public static final Comparator<Event> ORDER_BY_OPENING_HOURS = new Comparator<Event>() {
```

Comment:
In the example from week 5, the same convention was used: 
![image](https://user-images.githubusercontent.com/46226072/86623675-f6333100-bf8f-11ea-81b6-ef99c46f3468.png)
My understanding is that since comparators are only used for sorting, it makes sense to name them as SortBySomething? I also did a quick search on some examples online and changed the format to PascalCase.:D

Reply:
Ah okay. It's hard to argue when they set up the code like that.